util/encrypt.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) 2017-2019 The Decred developers
```

Comment:
Keys must always be pointers. You violate that rule in this file.

Reply:
Fixed.